src/system.c

```diff
@@ -1519,8 +1512,51 @@ static void SySetGapRootPath( const Char * string )
         n++;
 #endif
     }
+    return;
 }
 
+/****************************************************************************
+**
+*F  SySetInitialGapRootPaths( <string> )  . . . . .  set the root directories
+**
+**  Setup GAP's initial root paths, based on two sources:
+**  1) If the environment variable GAPROOT is undefined, the
+**     C macro GAPROOT_LOCATION (defined when GAP was compiled).
+**  2) Alternatively, try the location of GAP's executable.
+*/
+static void SySetInitialGapRootPaths(void)
+{
+
+#ifdef GAPROOT_LOCATION
+    SySetGapRootPath(GAPROOT_LOCATION);
+    return;
+#endif
+
+    if (GAPExecLocation[0] != 0) {
+        /* GAPExecLocation might be a super-directory of GAP root,
+         * so we will go and search for the true GAP root.
+         * By default we try stepping back 2 levels */
```

Comment:
"By default" makes it sound as if there was a way to override this, which there isn't, no?